pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -61,6 +62,8 @@ type kubenetNetworkPlugin struct {
 	mu          sync.Mutex //Mutex for protecting podCIDRs map and netConfig
 	execer      utilexec.Interface
 	nsenterPath string
+	wantPromisc bool
```

Comment:
Consider storing HairpinMode instead. Once the PR for hairpin support in cni bridge plugin gets in. 
We can just use it.